sentinelsat/sentinel.py

```diff
@@ -145,13 +144,13 @@ def query(self, area, initial_date=None, end_date=datetime.now(), **keywords):
         and any other search keywords accepted by the SciHub API.
         """
         query = self.format_query(area, initial_date, end_date, **keywords)
-        self.load_query(query)
-        return self.products
+        return self.load_query(query)
 
-    def load_query(self, query, start_row=0):
+    def load_query(self, query, start_row=0, products=[]):
         """Do a full-text query on the SciHub API using the format specified in
            https://scihub.copernicus.eu/twiki/do/view/SciHubUserGuide/3FullTextSearch
         """
+        output = products
```

Comment:
@lucadelu This is the reason why the tests are failing. You switch back and forth between the variables `products` and `outputs`. This is actually not necessary. Remove `outputs` and just keep appending to `products`. This requires that you call the extraction from`json_feed` something else in [line 171](https://github.com/ibamacsr/sentinelsat/pull/81/files#diff-b41f2cfc13d00858923dd3e8da963187L171).